Integrations/integration-McAfeeNSM.yml

```diff
@@ -0,0 +1,679 @@
+      default: true
+      description: the sensor ID related to the pcap
+    description: Get a pcap file of given event
+  - name: nsm-get-sensors
+    arguments:
+    - name: domainID
+      default: true
+      description: the domain of the sensors. If blank, returns all sensors.
+    outputs:
+    - contextPath: NSM.Sensors.ID
+      description: Sensor ID
+    - contextPath: NSM.Sensors.Description
+      description: Sensor Description
+    - contextPath: NSM.Sensors.DomainID
+      description: Sensor's Domain ID
+    description: get a sensor list of a given domain
+  - name: nsm-get-domains
+    arguments:
+    - name: domain
+      default: true
+      description: Specific domain details. Leave blank for all domains.
+    outputs:
+    - contextPath: NSM.Domains.ID
+      description: Domain ID
+    - contextPath: NSM.Domains.Name
+      description: Domain Name
+    description: Get all domains
+  - name: nsm-get-alerts
+    arguments:
+    - name: time_period
+      auto: PREDEFINED
+      predefined:
+      - LAST_5_MINUTES
+      - LAST_1_HOUR
+      - LAST_6_HOURS
+      - LAST_12_HOURS
+      - LAST_24_HOURS
+      - LAST_7_DAYS
+      - LAST_14_DAYS
+      - CUSTOM
+      description: Time Period
+    - name: start_time
+      description: Start Time in "mm/dd/yyyy HH:MM" format only. used for custom time
+        only
+    - name: end_time
+      description: End Time in "mm/dd/yyyy HH:MM" format only. used for custom time
+        only
+    - name: state
+      auto: PREDEFINED
+      predefined:
+      - ANY
+      - Acknowledged
+      - Unacknowledged
+      description: Alert State
+      defaultValue: ANY
+    - name: search
+      description: Search
+    - name: filter
+      description: 'Filter alert by fields. example: "name:hello;direction:Inbound,Outbound;attackcount:>3,<4".'
+    outputs:
+    - contextPath: NSM.Alerts.ID
+      description: Alert ID
+    - contextPath: NSM.Alerts.Name
+      description: Alert Name
+    - contextPath: NSM.Alerts.State
+      description: Alert State (Acknowledged,Unacknowledged)
+    - contextPath: NSM.Alerts.CreatedTime
+      description: Alert Creation Time
+    - contextPath: NSM.Alerts.Assignee
+      description: Alert Assignee
+    - contextPath: NSM.Alerts.AttackSeverity
+      description: Alert Severity
+    - contextPath: NSM.Alerts.Application
+      description: The Application assosiated to the alert
+    - contextPath: NSM.Alerts.EventResult
+      description: Event Result
+    - contextPath: NSM.Alerts.Event
+      description: The Event who triggered the alert
+    - contextPath: NSM.Alerts.Attack
+      description: Alert's Attack
+    - contextPath: NSM.Alerts.Attacker
+      description: The Attacker who committed the attack
+    - contextPath: NSM.Alerts.Target
+      description: The Attack's Target
+    - contextPath: NSM.Alerts.MalwareFile
+      description: Malware File used in the attack
+    description: Get alerts
+  - name: nsm-update-alerts
```

Comment:
Which alert does this command update?
What are the semantic of the alert state argument and the search argument?